aldryn_reversion/templates/aldryn_reversion/confirm_recover.html

```diff
@@ -0,0 +1,64 @@
+{% extends "admin/base_site.html" %}
+{% load i18n admin_urls %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} revert-confirmation{% endblock %}
+
+{% block breadcrumbs %}
+    <div class="breadcrumbs">
+        <a href="{% url 'admin:index' %}">{% trans "Home" %}</a> &rsaquo;
+        <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a> &rsaquo;
+        <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst|escape }}</a> &rsaquo;
+        <a href="{% url opts|admin_urlname:'change' object.pk|admin_urlquote %}">{{ object|truncatewords:"18" }}</a> &rsaquo;
+        <a href="../">{% trans "History" %}</a> &rsaquo;
+        {% blocktrans with opts.verbose_name as verbose_name %}Restore deleted {{ verbose_name }}{% endblocktrans %}
+    </div>
+{% endblock %}
+
+{% block content %}
+    <p>
+        {% blocktrans with escaped_object=object %}
+            Are you sure you want to restore the {{ object_name }} "{{ escaped_object }}" to its version of {{ revision_date }}?
```

Comment:
'Are you sure you want to restore the {{ object_name }} "{{ escaped_object }}" to its version **from** {{ revision_date }}'